pkgs/tools/system/auto-cpufreq/default.nix

```diff
@@ -1,17 +1,24 @@
-{ lib, python3Packages, fetchFromGitHub, substituteAll }:
+{ lib, python3Packages, fetchFromGitHub, substituteAll, wrapGAppsHook, gobject-introspection, gtk3 }:
 
 python3Packages.buildPythonPackage rec {
+  # use pyproject.toml instead of setup.py
+  format = "pyproject";
+
   pname = "auto-cpufreq";
-  version = "1.9.9";
+  version = "2.1.0";
 
   src = fetchFromGitHub {
     owner = "AdnanHodzic";
     repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21